lib/providers/quoters/RfqQuoter.ts

```diff
@@ -42,9 +48,15 @@ export class RfqQuoter implements Quoter {
       axios.get(`${this.serviceUrl}dutch-auction/nonce?address=${swapper}&chainId=${request.info.tokenInChainId}`), // should also work for cross-chain?
     ];
 
+    const getPortionResponse = await this.portionProvider.getPortion(
+      request.info,
+      mapNative(request.info.tokenIn, request.info.tokenInChainId),
+      mapNative(request.info.tokenOut, request.info.tokenOutChainId)
+    );
+
     let quote: Quote | null = null;
     metrics.putMetric(`RfqQuoterRequest`, 1);
-    await Promise.allSettled(requests).then((results) => {
+    await Promise.allSettled(requests).then(async (results) => {
```

Comment:
Why does this become async? Doesn't look like we've added any async calls in here?